src/activemasternode.cpp

```diff
@@ -154,29 +165,57 @@ void CActiveMasternode::ManageStateInitial()
         return;
     }
 
+    // Default to REMOTE
+    eType = MASTERNODE_REMOTE;
+
+    // Check if wallet funds are available
+    if(!pwalletMain) {
+        strNotCapableReason = "Wallet not available.";
+        LogPrintf("CActiveMasternode::ManageStateInitial -- not capable: %s\n", strNotCapableReason);
+        return;
+    }
+
+    if(pwalletMain->IsLocked()) {
+        strNotCapableReason = "Wallet is locked.";
+        LogPrintf("CActiveMasternode::ManageStateInitial -- not capable: %s\n", strNotCapableReason);
+        return;
+    }
+
+    if(pwalletMain->GetBalance() < 1000*COIN) {
```

Comment:
In the current version this check is only done once.  Once the ACTIVE_MASTERNODE_STARTED state is reached no more checks are performed and the pinger always runs.